zoo/alexnet.py

```diff
@@ -0,0 +1,182 @@
+###############################################################################
+# .. attention::
+#
+# Copyright(c) 2021 Francesco Loro, Master Degree Student Università degli studi di Padova.
+# All rights reserved.
+#
+# This software component is licensed by Apache License Version 2.0
+# http://www.apache.org/licenses/
+# Same as QKeras
+# You may not use this file except in compliance with# the License.
+#
+###############################################################################
+
+__author__ = "Francesco Loro"
+__email__ = "francesco.official@gmail.com"
+__supervisor__ = "Danilo Pau"
+__email__ = "danilo.pau@st.com"
+
+# Download pretrained weight from:
+# Alexnet -> https://github.com/larq/zoo/releases/download/binary_alexnet-v0.2.0/binary_alexnet_weights.h5
+
+import qkeras as q
+import tensorflow as tf
+import larq as lq
+from utils import compare_network, create_random_dataset, dump_network_to_json
+
+# Define path to the pre-trained weights
+path_alexnet = "./weigths/binary_alexnet_weights.h5"
+alexnet_name = "alexNet"
+
+
+class alexnet():
```

Comment:
remove "()"
and also follow https://google.github.io/styleguide/pyguide.html#3162-naming-conventions